### PR TITLE
[PDE-3809] chore: Replace issue templates with GitHub forms, update CODEOWNERS

### PR DESCRIPTION
**Pull request by rnegron**
Changes in this PR, with explanations:
* Updated `CODEOWNERS` file to remove former Zapien David Brownman. Added myself to keep 2 reviewers!
* Removed the `question.md` issue template since we don't want to funnel general questions through GitHub issues.
* Added `config.yml` to issue template folder, providing some extra contact links and preventing blank issues from being created. More info: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
* Replaced `bug-report.md` issue template with `bug-report.yml`, a GitHub form.
* Replaced `feature-request.md` with `feature-report.yml`, a GitHub form.

More info on Issue template forms: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

### CODE REVIEW

**Review by eliangcs — DISMISSED** [review]
🎉

**Comment by rnegron** [comment]
Thank you for the PR approval @eliangcs! I noticed some formatting issues before merging, which after fixing dismissed your review, sorry!

**Review by eliangcs — APPROVED** [review]

**Pull request merged**
